QUEST_LV_0200.tsv

```diff
@@ -7739,87 +7739,87 @@ QUEST_LV_0200_20150714_007742	I should've persuaded him slowly, but I think we w
 QUEST_LV_0200_20150714_007783	That's it. I am sorry, but can you help me with one more thing?{nl}If I only have the resin that is collected in the head of Wurudwa, I think I will be able to resolve it.
 QUEST_LV_0200_20150714_007784	I didn't expect I would meet someone other than those hunters.{nl}I am sure the Goddesses haven't abondoned me yet.
-QUEST_LV_0200_20150714_007785	I wasn't able to do anything myself, but it is so fortunate that the Goddesses have sent you.{nl}I will make a medicine. Thank you so much!
+QUEST_LV_0200_20150714_007785	I wasn't able to do anything myself, but it is so fortunate that the Goddesses have sent you.{nl}I will make some medicine. Thank you so much!
 QUEST_LV_0200_20150714_007786	Stranger...{nl}If you don't have any task here, please go back.
 QUEST_LV_0200_20150714_007787	The monastery is locked with the barriers of the monks so it won't be possible without the persuasion from Guus.{nl}But, I... it's too difficult.
 QUEST_LV_0200_20150714_007788	Hunters... they just won't give up.{nl}I've heard Marko and Potos are with the hunters.{nl}
 QUEST_LV_0200_20150714_007789	But, you should be cautious.{nl}We just can't listen to one side.
-QUEST_LV_0200_20150714_007790	After persuading him, we can immediately start to retrieve the monastery.{nl}We can expect to expel Evoniphon at then.
+QUEST_LV_0200_20150714_007790	After persuading him, we can immediately start to retrieve the monastery.{nl}We can expect to expel Evoniphon then.
 QUEST_LV_0200_20150714_007791	Without Guus, we won't be able to recover the monastery.{nl}I hope you succeed in persuading him.
 QUEST_LV_0200_20150714_007792	But, I think we can resume the talk after retrieving the monastery.{nl}Please show me the competence which I could rely on so I can cooperate with you.
-QUEST_LV_0200_20150714_007793	Good.{nl}The Liverworts and Stonakons aroud here would be enough. Please also bring the proofs.
+QUEST_LV_0200_20150714_007793	Good.{nl}The Liverworts and Stonakons around here would be enough. Please also bring the proofs.
 QUEST_LV_0200_20150714_007794	You can give up if you feel that's too much for you.{nl}We were gonna finish it ourselves.
 QUEST_LV_0200_20150714_007795	You definitely possess the competence. We also have the same objective.{nl}There is no reason to insist anymore.
 QUEST_LV_0200_20150714_007796	The senior monk here is very picky.
-QUEST_LV_0200_20150714_007797	I will gather the monks in front of the entrance of the monastery so tell hunters to gather as well.{nl}After they are gathered, we will unleash the seals that are blocked so that the monsters won't be able to follow.{nl}
+QUEST_LV_0200_20150714_007797	I will gather the monks in front of the entrance of the monastery so tell hunters to gather as well.{nl}After they are gathered, we will unleash seals to block any monsters from following us.{nl}
```

Comment:
I will gather the monks in front of the entrance of the monastery so tell hunters to gather as well.{nl}After they are gathered, we will unleash seals to block any monsters from following us.{nl}

to

I will gather the monks in front of the entrance of the monastery, so tell the Hunters to gather as well.{nl}After they are here, we will unleash the seals to block any monsters from following us.{nl}